schema/deploy/shipping/views.sql

```diff
@@ -1172,6 +1173,10 @@ create or replace view shipping.scan_encounters_v1 as
         age_in_years(lower(age_bin_coarse_v2.range)) as age_range_coarse_lower,
         age_in_years(upper(age_bin_coarse_v2.range)) as age_range_coarse_upper,
 
+        age_bin_decade_v1.range as age_range_decade,
+        age_in_years(lower(age_bin_decade_v1.range)) as age_range_decade_lower,
+        age_in_years(upper(age_bin_decade_v1.range)) as age_range_decade_upper,
```

Comment:
I'm still learning about postgres ranges, and so I'm wondering why we return all three of these values. If we include the range itself, does that contain the same information as pulling out the lower and upper explicitly?

Reply:
We've gotten feedback in the past from downstream users (I think IDM?) that the "range" itself can be tricky to use in certain cases, so it would be best to also include the int version of the ranges.